README.md

```diff
@@ -316,8 +320,58 @@ The Realm Dart package is `realm_dart`
 
     _*This file should be committed to source control_
 
-* For more usage of Realm Dart see the Realm Flutter usage above.
+* For more information on using the Realm Dart Standalone SDK see the Realm Flutter SDK usage above.
+
+# Device Sync with Realm Flutter and Dart Standalone SDKs
+
+Run the Realm with [Device Sync](https://www.mongodb.com/docs/realm/sdk/flutter/sync/) and connecting to [Atlas App Services](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/).
```

Comment:
What does it mean to `Run the Realm`? I think we need to rephrase it here not sure how since it is not clear to me what is the purpose of this sentence? Provide the links to the docs?

Reply:
Edited.